### PR TITLE
add adopters.md

### PR DESCRIPTION
**Pull request by dovholuknf**
This will coincide with a discourse post for the same. initial doc here to see what we think

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Adopters
```
Comment (dovholuknf) [inline ADOPTERS.md:1]: Rendered:

image

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+This is a page listing the adopters of the OpenZiti project along with a brief description of what the project/product offers.
+
+## Adding Your Project to the List
+
+If you're interested in getting your project added to the list either:
```
Comment (smilindave26) [inline ADOPTERS.md:7]: "either" seems wrong here

**Review by smilindave26 — APPROVED** [review]

File: ADOPTERS.md (hunk 2 above)
Comment (dovholuknf) [reply ADOPTERS.md:7]: i forgot to add "put up a pull request editing the table below". thx i will fix that.

Comment (dovholuknf) [reply ADOPTERS.md:7]: added to bullets

**Pull request merged**